pycomfoconnect/comfoconnect.py

```diff
@@ -351,9 +399,10 @@ def _connect(self, takeover=False):
             self.cmd_register_app(self._local_uuid, self._local_devicename, self._pin, use_queue=False)
 
             # Login
-            self.cmd_start_session(takeover, use_queue=False)
-
-        return True
+            message = self.cmd_start_session(takeover, use_queue=False)
+        
+        return (message)
```

Comment:
This returns a tuple? Next line should be removed.

Reply:
To allow parsing of reply in the calling function(s)

Reply:
Sure, then you just need a return message without parentheses